DefaultConfigsExtension/src/main/java/de/viadee/xai/anchor/adapter/tabular/discretizer/impl/AmevaDiscretizer.java

```diff
@@ -52,7 +52,7 @@ public AmevaDiscretizer() {
         }
 
         keyValuePairs = IntStream.range(0, values.length)
-                .mapToObj(i -> new AbstractMap.SimpleImmutableEntry<>((Double) values[i], labels[i]))
+                .mapToObj(i -> new AbstractMap.SimpleImmutableEntry<>(((Number) values[i]).doubleValue(), labels[i]))
```

Comment:
Why the generalization / downcast?

Reply:
If all values were of Integer, a castingexception would occur, because casting from Integer to Double ( values[i] gives Integer), is not possible.

Reply:
After the doublevalue of the Number-Type is taken, it will be converted to Double, i did not change the key and value Types of the AbstractMap